1_fetch.R

```diff
@@ -11,6 +11,9 @@ source("1_fetch/src/fetch_nhdv2_attributes_from_sb.R")
 source("1_fetch/src/download_file.R")
 source("1_fetch/src/munge_reach_attr_tbl.R")
 
+# tar_cue for downloading NWIS sites and data
```

Comment:
I understand wanting peace of mind that the NWIS targets haven't been rebuilt without our realizing it. If we _did_ decide to re-build those targets, what would we change `NWIS_cue` to? `always`? A small note in the comment here might be useful for indicating that.

Reply:
Thanks, I'll add that comment. It would change to 'thorough'